### ISSUE TITLE
User Survey Request for ORMDI

### ISSUE THREAD

**Issue by jwoodman5** [department-of-veterans-affairs/va.gov-team#18319]
# Request the Medallia Survey

## What this form is for
Use this template to request adding the Medallia Survey to your product.

Read more about Medallia [link](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/platform/medallia) & how to request a survey [link](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/medallia-request-survey.md)

## Description
A new Medallia survey needs to be added to 8 pages listed below. Staging, of course, will be first.

### Acceptance Criteria
- Survey script provided below is added to the 8 _staging_ pages/URLs listed below.

### Launch Date
*January 22nd, however, survey implementation by Feb. 1st is okay.*

### Survey to Implement (script)
- Approval is attached below
- <script type="text/javascript" src="https://resource.digital.voice.va.gov/wdcvoice/5/onsite/embed.js" async></script>

### Staging URLs
- https://www.staging.va.gov/ORMDI/
- https://www.staging.va.gov/ORMDI/NoFEAR_Select.asp
- https://www.staging.va.gov/ORMDI/Contact_Us.asp
- https://www.staging.va.gov/ORMDI/EEOcomplaint.asp
- https://www.staging.va.gov/ORMDI/HPP.asp
- https://www.staging.va.gov/ORMDI/Diversity_Inclusion.asp
- https://www.staging.va.gov/adr/
- https://www.staging.va.gov/ORMDI/Reasonable_Accommodations1.asp

### Production URLs
- https://www.va.gov/ORMDI/
- https://www.va.gov/ORMDI/NoFEAR_Select.asp
- https://www.va.gov/ORMDI/Contact_Us.asp
- https://www.va.gov/ORMDI/EEOcomplaint.asp
- https://www.va.gov/ORMDI/HPP.asp
- https://www.va.gov/ORMDI/Diversity_Inclusion.asp
- https://www.va.gov/adr/
- https://www.va.gov/ORMDI/Reasonable_Accommodations1.as

---

*Please leave the following blank*

---

## Acceptance Criteria
- [ ] URLs added & QA'd with the Medallia survey

**Comment by jwoodman5** [comment]
Survey Approval
[ORMDI Brand Health OMB Approval.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/5789538/ORMDI.Brand.Health.OMB.Approval.pdf)

**Comment by billfienberg** [comment]
It looks like the eight pages are all `.asp` pages, and live outside of the `vets-website` repo. 

I see that Foresee analytics are being loaded on `/ORMDI`. 

![image](https://user-images.githubusercontent.com/6130520/106489439-65b3c500-647a-11eb-9fc7-c8fd38f07296.png)

I'm currently studying the `proxy-rewrite` [application](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications/proxy-rewrite) for prior art.

**Comment by billfienberg** [comment]
It looks like both the [staging](https://staging.va.gov/ORMDI/) and [prod](https://va.gov/ORMDI/) versions of `/ORMDI` are using the production Foresee [script](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/assets/js/foresee/vagovprod.js). 

![image](https://user-images.githubusercontent.com/6130520/106506694-acf88080-648f-11eb-9d78-5fc63dee93bf.png)

**Comment by jwoodman5** [comment]
@billfienberg I don't know the answer to your question directly, but do know, I should remove the staging URLs from ForeSee and I'll start working on that.

**Comment by billfienberg** [comment]
> I don't know the answer to your question directly, but do know, I should remove the staging URLs from ForeSee and I'll start working on that.

@jwoodman5, sorry for any confusion. I didn't have a specific question. I was just using issue comments as places to document some findings. If I have specific questions, I'll try to tag you in the comment and/or in Slack.

**Comment by billfienberg** [comment]
## Enterprise Web Infrastructure Support (EWIS)

### Example EWIS Page

https://www.va.gov/health/

![image](https://user-images.githubusercontent.com/6130520/106518421-7b87b100-649f-11eb-8ca6-e13847dd416e.png)

### Links

- https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/architecture/teamsite/
- https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/VA-Systems/EWIS/ewis-proxy-details.md

**Comment by billfienberg** [comment]
I think I figured out why Foresee was showing up, and Medallia isn't. 

TLDR: I think EWIS pages use `vagovprod`, and the Medallia script is currently only rendered in the `vagovstaging` environment. 

### Links 

- https://github.com/department-of-veterans-affairs/vets-website/blob/8b64d81ec200805e9fbc833a53698c39fed6ec1b/src/site/includes/survey-tools.html#L1-L12

**Comment by billfienberg** [comment]
Conversation with @jwoodman5 in Slack
>**Bill**: Are we ready to enable Medallia in prod yet? Or do we still only want it on staging for now? Or do we want it on in prod for some pages, but only staging for other pages?
>I was under the impression that we’re not ready to enable Medallia in prod on `/find-locations` and `/find-locations/facility/vha_691`.
>
>**Jason**: Correct, not ready to enable on prod until we get those sev-1 508 fixes done. But as soon as we are, those pages will be first.
>
>**Bill**: That answer is relevant to the code that I’m writing because EWIS/TeamSite don’t seem to differentiate between prod and staging, so I’m trying to handle those environment differences in my code.

**Comment by billfienberg** [comment]
Difference between https://staging.va.gov/health/ (`vagovprod`) and https://staging.va.gov/health/?targetEnvironment=vagovstaging (`vagovstaging`)

![proxy-rewrite](https://user-images.githubusercontent.com/6130520/106528683-c52bc800-64ae-11eb-9771-f3aa01530cf7.png)

https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/architecture/teamsite#local-dev
